src/backend/k3sHelper.ts

```diff
@@ -940,7 +940,7 @@ export default class K3sHelper extends events.EventEmitter {
    * deleted.
    * @param execAsRoot A function to run commands on the VM as root.
```

Comment:
This @param is now obsolete

Reply:
Fixed in 7994a25a8e04a7f0435170bfabd408e23ffbc116.